clang/test/Driver/sycl-early-device-link.cpp

```diff
@@ -0,0 +1,71 @@
+// Testing for early AOT device linking.  These tests use -fno-sycl-rdc
+// -c to create final device binaries during the link step when using -fsycl.
+// Behavior is restricted to spir64_gen targets for now.
+
+// Create object that contains final device image
+// RUN: %clangxx -c -fno-sycl-rdc -fsycl -fsycl-targets=spir64_gen \
+// RUN:          --target=x86_64-unknown-linux-gnu -Xsycl-target-backend \
+// RUN:          "-device skl" --sysroot=%S/Inputs/SYCL -### %s 2>&1 \
+// RUN:  | FileCheck %s -check-prefix=CREATE_IMAGE
+// CREATE_IMAGE: clang{{.*}} "-triple" "spir64_gen-unknown-unknown"{{.*}} "-fsycl-is-device"{{.*}} "-o" "[[DEVICE_BC:.+\.bc]]"
+// CREATE_IMAGE: llvm-link{{.*}} "-o" "[[DEVICE_BC2:.+\.bc]]"
```

Comment:
is DEVICE_BC2 the sycl device library link? If so could we rename the var?

Reply:
Yes, it is the device library link - I will update the names to better match file content expectations.